Makefile

```diff
@@ -86,7 +86,7 @@ test-integration:
 endif
 
 .PHONY: build
-build: build-ubuntu
+build: build-ubuntu build-ubi
```

Comment:
can we avoid adding the new `build-ubi` target to the "default" target (`build` / `all`)? When a developer runs `make`, they probably only want the ubuntu image and don't want to have to wait for both builds

Reply:
Done. Remove `build-ubi` target from default target.